app/resources/admin.html.template

```diff
@@ -138,559 +167,616 @@
     var title_col = row.insertCell(cur_col++);
     var title = document.createElement('input');
     title.type = 'text';
-    title.id = 'language_title_' + iteration;
+    title.id = table_id + '_language_value_' + iteration;
     title_col.appendChild(title);
-
     return iteration;
   }
 
+  function add_language_span(table_id, language) {
+    var table = $(table_id + '_table');
+    iteration = table.rows.length;
+    var row = table.insertRow(iteration);
+    var cur_col = 0;
+  
+    var selected_language_col = row.insertCell(cur_col++);
+    var selected_language = document.createElement('span');
+    var selected_language_text = document.createTextNode(language);
+    selected_language.className = 'language_selected_' + iteration;
+    selected_language.appendChild(selected_language_text);
+    selected_language_col.style.verticalAlign = 'top';
+    selected_language_col.appendChild(selected_language);
+
+    var title_col = row.insertCell(cur_col++);
+    var title = document.createElement('textarea');
+    title.cols = 72;
+    title.id = table_id + '_language_value_' + iteration;
+    title_col.appendChild(title);
+    return iteration; 
+  }
+
+  function add_another_language() {
```

Comment:
It's too confusing to have add_another_language() and add_language(). Maybe rename add_language() to a clearer name e.g., add_language_row_with_drop_down()?